README.md

```diff
@@ -19,26 +19,24 @@
 
 ---
 
-## Development
+## Build docs locally
 
-1. Checkout this repository
-1. Install mintlify with `npm i -g mintlify@latest`
-1. Run `mintlify dev`
+1. Fork this repository.
+1. Install mintlify with `npm i -g mintlify@latest`.
+1. Run `mintlify dev`.
 
 ## Deploying 
 
-To build the final product, we have a couple of additional steps:
+To build the final product:
 
-1. Build the JS Protocol Docs
+1. Build the JS Protocol Docs:
 
 - `npm install -g jsdoc-to-markdown`
```

Comment:
As above, where do you run this? And do they have to be separate commands, or can you run them together? If you can run them together, then why aren't they in a code block?